de.hamstersimulator.objectsfirst.ui/src/main/java/de/hamstersimulator/objectsfirst/ui/javafx/GameSceneController.java

```diff
@@ -19,24 +19,53 @@
 import javafx.scene.layout.BorderPane;
 import javafx.scene.paint.Color;
 
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.logging.LogManager;
+
+/**
+ * Controller for the main game scene
+ *
+ * Note: The static constructor sets the log level of the javafx.scene.control logger to WARNING to
+ * hide a INFO message caused by the log list
+ */
 public class GameSceneController {
 
+    static {
+        final String logLevelString = "javafx.scene.control.level = WARNING";
+        final InputStream logLevelStream = new ByteArrayInputStream(logLevelString.getBytes(StandardCharsets.UTF_8));
+        final Function<String, BiFunction<String, String, String>> mapper = (category) -> {
+            return (oldValue, newValue) -> {
+                return Optional.ofNullable(newValue).orElse(oldValue);
+            };
+        };
+
+        try {
+            LogManager.getLogManager().updateConfiguration(logLevelStream, mapper);
+        } catch (IOException e) {
+            // Do nothing, in this case we failed to update the log level
```

Comment:
Might be good to log this issue then... hoping it does not happen :)

Reply:
fixed with ba62fece1bb282a571487ef6f41f112bc51f847a